nerfstudio/models/splatfacto.py

```diff
@@ -668,11 +674,13 @@ def get_outputs(self, camera: Cameras) -> Dict[str, Union[torch.Tensor, List]]:
                 background = self.background_color.to(self.device)
         else:
             optimized_camera_to_world = camera.camera_to_worlds[0, ...]
-
-            if renderers.BACKGROUND_COLOR_OVERRIDE is not None:
-                background = renderers.BACKGROUND_COLOR_OVERRIDE.to(self.device)
+            if self.config.background_color == "auto":
+                background = (torch.sigmoid(self.learnable_bg).to(self.device) * 1.1).clamp(0, 1)
```

Comment:
what is this 1.1 multiplier for?

Reply:
sigmoid itself can hardly produce pure white (also pure black), like those for blender dataset, or overexposed images. So 1.1 just allows it to generate pure white